libs/hmpb/src/ballot_templates/vx_default_ballot_template.tsx

```diff
@@ -548,6 +550,17 @@ async function BallotPageContent(
     );
   }
 
+  const contestsLeftToLayout = contestSectionsLeftToLayout.flat();
+  if (
+    contests.length > 0 &&
+    contestsLeftToLayout.flat().length === contests.length
+  ) {
+    return err({
+      error: 'contestTooLong',
+      contest: contestsLeftToLayout[0],
+    });
+  }
```

Comment:
factor out to helper?

Reply:
I feel like there's a larger factoring out needed for contest layout logic that would include this, but I don't have time to do it now and I don't think there's much to be gained just from factoring out these few lines, since they only make sense in context